src/chaos-app.js

```diff
@@ -10,8 +10,9 @@ Tactics.App = (function ($, window, document) {
     {assignment:[5, 1], type:'Knight'},
     {assignment:[6, 1], type:'Knight'},
   ];
-  var gameData = {
```

Comment:
@latin-programmer Nesting the data inside of a "data" tag shouldn't be necessary and should be reverted.  The so-called "gameData" variable would probably be better named as "gameStateData" since it will ultimately be consumed by the "tactics/GameState.js" class when loading an instance.